Source/ACE.Server/WorldObjects/Managers/EmoteManager.cs

```diff
@@ -1218,6 +1209,124 @@ public float ExecuteEmote(BiotaPropertiesEmote emoteSet, BiotaPropertiesEmoteAct
             return delay;
         }
 
+        private static void CheckForQuestManager(Creature creature)
+        {
+            if (creature != null && creature.QuestManager == null)
+                creature.QuestManager = new QuestManager(creature);
+        }
+
+        private void UpdateQuest(BiotaPropertiesEmoteAction emote, WorldObject targetObject, Creature creature)
```

Comment:
EmoteManager should not contain special methods for particular emotes... if these needed to be methods, they belong in QuestManager

However, there is no need for these to be separate methods